app/views/layouts/partials/_footer.html.erb

```diff
@@ -0,0 +1,16 @@
+<footer class="footer w-full bg-gray-900 ">
+    <div class="container mx-auto text-white text-center pt-5">
+        <h3>Team:</h3>
+        <ul class="list-none">
+            <li>Heriberto</li>
+            <li>Juan Ramon</li>
+            <li>Jose Maria</li>
+            <li>Adrian</li>
+        </ul>
+    </div>
+    <div class="container mx-auto my-4 py-4 text-center">
+        <div class="border-t border-white w-6/12 mx-auto pt-5">
+            <p class="text-white"><a href="http://www.brightcoders.com/">BrightCoders</a> | 2021 </p>
+        </div>
+    </div>
+</footer>
```

Comment:
keep your indentation with 2 spaces

Reply:
Thanks for your review, i already change the indentation with 4 spaces to 2 spaces.